sdk/storage/Azure.Storage.Queues/src/QueueClient.cs

```diff
@@ -2043,5 +2092,59 @@ private void SetNameFieldsIfNull()
             }
         }
         #endregion UpdateMessage
+
+        private async Task<string> ClientSideEncryptInternal(string messageToUpload, bool async, CancellationToken cancellationToken)
```

Comment:
This may not be too important since the queue stuff is rarely touched, but you could consider moving these methods to a separate class to avoid cluttering the queue client.

Reply:
addressed in upcoming push